catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/SeedCommand.java

```diff
@@ -69,10 +60,11 @@
                 + "This command will not re-download resources that are up-to-date in the cache, so subsequent runs "
                 + "of the command will attempt to cache metacards and resources that have not already been cached.\n\n"
                 + "Note: this command will trigger resource downloads in the background and they may continue after "
-                + "control is returned to the console. Also, resource caching must be enabled in the catalog framework "
+                + "control is returned to the console. Also, resource caching must be enabled in the Catalog Framework "
                 + "for this command to seed the resource cache.")
 //@formatter:on
-public class SeedCommand extends SubjectCommands implements Action {
+public class SeedCommand extends CqlCommands {
```

Comment:
It should be fine to have them. The only issue is that this command won't work with the `--provider` option because it needs to be run through the framework.